ui/common/StartButton.tsx

```diff
@@ -91,20 +97,27 @@ export default function Card({
   }
 
   return (
-    <div className="order-last flex items-center justify-center text-[15px] font-bold">
-      <div
-        className={clsx(
-          'order-last mr-2.5 flex h-[29px] w-[44px] cursor-pointer items-center justify-center  rounded-[3px] border-2 ',
-          {
-            'border-black/25 text-black/25': powerUp === false,
-            ' animate-duration-500 animate-pulse border-[#FBEBC6] bg-[#FBEBC6] text-[#571A1F] shadow-[0_0px_15px_rgba(251,235,198,0.75)]':
-              powerUp === true,
-          }
-        )}
-        onClick={handleClick}
-      >
-        10x
-      </div>
+    <div
```

Comment:
Let's use the `<button>` for buttons (or anchor if it's for navigation). Better for accessibility and you get button-like behavior for free (just need to remove the default styling).